irohad/model/execution/impl/command_executor.cpp

```diff
@@ -60,7 +60,7 @@ bool AddAssetQuantityExecutor::execute(const Command &command,
   }
   auto precision = asset.value().precision;
 
-  if (add_asset_quantity.amount.get_frac_number() > precision) {
+  if (add_asset_quantity.amount.getPrecision() != precision) {
     log_->info("amount is wrong formed:");
```

Comment:
I think "wrongly" fits better, or malformed

Reply:
Not mine code, but ok